core/string/string_name.cpp

```diff
@@ -59,6 +63,24 @@ void StringName::setup() {
 void StringName::cleanup() {
 	MutexLock lock(mutex);
 
+#ifdef DEBUG_ENABLED
+
+	if (debug_stringname) {
```

Comment:
Should this use `unlikely()` too?

Reply:
i added it, though its only called once so it should not be a problem